core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -217,6 +217,15 @@ public LockResource lockExclusive(StateLockOptions lockOptions)
     });
   }
 
+  /**
+   * Used to generate a call-tracker that knows when an interrupt cycle is active.
+   *
+   * @return the call tracker
+   */
+  public CallTracker getInterruptCycleTracker() {
+    return () -> mInterruptCycleRefCount > 0;
```

Comment:
Does this create a new CallTracker each time? Is there a way to have a single instance of the call tracker?

Reply:
singleton now.